parsers/s01-parse/crowdsecurity/apache2-logs.md

```diff
@@ -1 +1,3 @@
 This apache2 parser support access and error logs in the HTTPD COMBINED LOG standard format.
+
+*note : * If you are aggregating logs from several domains, prefix your logline with the target FQDN. HTTP based scenarios should take this into account so that buckets are _per_ source IP per target FQDN, limiting false positives due to logs multiplexing.
```

Comment:
Isn't better to prevent the users that the ram consumption can be impacted by the number of customers they can have, (many customers = more ram consumption).